### PR TITLE
drtprod: add init/run script for tpch workload

### PR DESCRIPTION
**Pull request by shailendra-patel**
Changes as part of this PR:
- Added `generate_tpch_init.sh` script for import tpch workload.
- Added `generate_tpch_run.sh` script for running tpch workload.
- Modified `tpcc_init.sh` to use fixtures for importing data.
- Added `tpcc_run_multiregion.sh` to run tpcc workload on multinode cluster.
- Modified `drt-large.yaml` and `drt-scale.yaml` to use above changes.

Epic: none

Release note: None

### CODE REVIEW

**Review by nameisbhaskar — APPROVED** [review]
LGTM! Just one minor comment to ensure that somebody executing the script directly knows what is required to be set.

**Review by vidit-bhat — APPROVED** [review]

File: pkg/cmd/drtprod/scripts/tpcc_run_multiregion.sh [hunk 1]
```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+# Copyright 2024 The Cockroach Authors.
+#
+# Use of this software is governed by the CockroachDB Software License
+# included in the /LICENSE file.
+
+# This script sets up the tpcc multiregion run configuration on the workload node.
+set -eux
+
```
[the remark this answers is not in the document]
Comment (shailendra-patel) [reply pkg/cmd/drtprod/scripts/tpcc_run_multiregion.sh:16]: Done

**Comment by shailendra-patel** [comment]
TFTR!

bors r = vidit-bhat, nameisbhaskar

**Pull request merged**